frank/fit.py

```diff
@@ -492,18 +617,14 @@ def perform_bootstrap(u, v, vis, weights, geom, model):
         logging.info(' Bootstrap trial {} of {}'.format(trial + 1,
                                                         model['analysis']['bootstrap_ntrials']))
 
-        u_s, v_s, vis_s, w_s = \
-            utilities.draw_bootstrap_sample(u, v, vis, weights)
+        u_s, v_s, vis_s, w_s = utilities.draw_bootstrap_sample(
+            u, v, vis, weights)
 
         sol, _ = perform_fit(u_s, v_s, vis_s, w_s, geom, model)
 
-        if model['analysis']['bootstrap_non_negative']:
-            profiles_bootstrap.append(sol.solve_non_negative())
-        else:
-            profiles_bootstrap.append(sol.mean)
+        profiles_bootstrap.append(sol.mean)
```

Comment:
86093bf appends `sol._nonneg`  (rather than `sol.mean`) to `profiles_bootstrap`  if `model['hyperparameters']['nonnegative']` is True. This now doesn't alter the `sol._mu` attribute (see in relation to your comment above).